net/tcp/tcp_sendfile.c

```diff
@@ -87,17 +86,63 @@ struct sendfile_s
   uint32_t           snd_acked;            /* The number of bytes acked */
 };
 
+/****************************************************************************
+ * Private Function Prototypes
+ ****************************************************************************/
+
+static inline void sendfile_txnotify(FAR struct socket *psock,
```

Comment:
move the sendfile_txnotify() implement before caller

Reply:
I did not do that to reduce number of lines relocations (avoid the whole sendfile_txnotify() function relocation) to simplify the source code diff / git history.
Though, I have done this now as you requested.